plugins/topology/src/utils/pod-resource-utils.test.ts

```diff
@@ -1,6 +1,32 @@
-import { getDeploymentRevision } from './pod-resource-utils';
+import {
+  V1Deployment,
+  V1StatefulSet,
+  V1DaemonSet,
+  V1CronJob,
+} from '@kubernetes/client-node';
+import { mockKubernetesResponse } from '../__fixtures__/1-deployments';
+import {
+  getPodsForStatefulSet,
+  getPodsForDaemonSet,
+  getPodsForCronJob,
+  getDeploymentRevision,
+  getPodsForDeployment,
+} from './pod-resource-utils';
 
 describe('PodResourceUtils', () => {
+  let mockResources = {};
+  beforeEach(() => {
+    mockResources = {
+      deployments: mockKubernetesResponse.deployments,
+      pods: { data: mockKubernetesResponse.pods },
+      jobs: { data: mockKubernetesResponse.jobs },
+      cronJobs: mockKubernetesResponse.cronJobs,
+      daemonSets: mockKubernetesResponse.daemonSets,
+      statefulsets: { data: mockKubernetesResponse.statefulsets },
+      replicasets: { data: mockKubernetesResponse.replicasets },
+    };
```

Comment:
Should this be as below?

```suggestion
    mockResources = {
      deployments:  { data: mockKubernetesResponse.deployments },
      pods: { data: mockKubernetesResponse.pods },
      jobs: { data: mockKubernetesResponse.jobs },
      cronJobs:  { data: mockKubernetesResponse.cronJobs },
      daemonSets:  { data: mockKubernetesResponse.daemonSets },
      statefulsets: { data: mockKubernetesResponse.statefulsets },
      replicasets: { data: mockKubernetesResponse.replicasets },
    };
```